game/guitargraph.cc

```diff
@@ -82,9 +83,9 @@ void GuitarGraph::initDrums() {
 	// Copy all tracks of drum type to m_instrumentTracks
 	for (auto const& elem: m_song.instrumentTracks) {
 		std::string index = elem.first;
-		if (index == TrackName::DRUMS 
-			|| index == TrackName::DRUMS_SNARE 
-			|| index == TrackName::DRUMS_CYMBALS 
+		if (index == TrackName::DRUMS
+			|| index == TrackName::DRUMS_SNARE
+			|| index == TrackName::DRUMS_CYMBALS
```

Comment:
|| and && go at the end of the line, not the beginning of the line.

so:

if (x == TrackName::Drums &&
    x == TrackName::Drums_SNARE &&
    x == TrackName::Drums_TOMS) {
}